http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -197,7 +197,10 @@ object clusterQuery extends TableQuery(new ClusterTable(_)) {
   //   left join label l on c.id = l.clusterId
   val clusterLabelQuery: Query[(ClusterTable, Rep[Option[LabelTable]]), (ClusterRecord, Option[LabelRecord]), Seq] = {
     for {
-      (cluster, label) <- clusterQuery joinLeft labelQuery.runtimeLabels on (_.id === _.resourceId)
```

Comment:
I removed `labelQuery.runtimeLabels` -- I think it's more efficient to do the `lbl.resourceType === LabelResourceType.runtime` filter in the join as opposed to a separate inner query.